docs/contributing/Regular-Expression-Regex.html

```diff
@@ -0,0 +1,1140 @@
+  <li>The character <code>\d</code> is used to match the integer numbers 0 to 9.</li>
+  <li>The <code>+</code> means <i>one or more times</i>.</li>
+  <li>The Regex therefore finds <code>RichMerchantsSkyrim_x</code> followed by one or more integer numbers.</li>
+</ul>
+
+
+<h3>Exactly x times: <code>{x}</code></h3>
+
+<p>Exemplary masterlist entry: <a href="https://www.nexusmods.com/oblivion/mods/23783/">Enhanced Vegetation</a></p>
+
+<pre><code>  - name: 'Enhanced Vegetation \[\d{3}%\]\.esp'
+    url: [ 'https://www.nexusmods.com/oblivion/mods/23783/' ]
+    msg:
+      - <<: *useOnlyOneX
+        subs: [ 'Enhanced Vegetation' ]
+        condition: 'many("Enhanced Vegetation \[\d{3}%\]\.esp")'
+</code></pre>
+
+<p>Plugins that are identified by the above Regex:</p>
+
+<pre><code>Enhanced Vegetation [100%].esp
+Enhanced Vegetation [110%].esp
+Enhanced Vegetation [125%].esp
+Enhanced Vegetation [150%].esp
+</code></pre>
+
+<p>Summary:</p>
+
+<ul>
+  <li>The period <code>.</code> has been escaped through writing a backslash <code>\</code> in front of it.</li>
+  <li>The square brackets <code>[</code> and <code>]</code> have been escaped as well, since otherwise they would be interpreted as a regular expression.</li>
+  <li>Within the square brackets is the expression <code>\d{3}</code> which meaning becomes <i>exactly three integer numbers</i>.</li>
+  <li>After this expression then comes a <code>%</code>, which is a normal character and which is needed because the plugin names include it.</li>
+  <li>Notice that besides for <code>name</code> the regex can also be used for <code>condition</code>.</li>
+</ul>
+
+
+<h3>x or more times: <code>{x,}</code></h3>
+
+<p>Exemplary masterlist entry:</p>
+
+<pre><code>  - name: 'Example \d{3,}\.esp'
+    msg:
+      - type: say
+        content:
+          - lang: en
+            text: 'A few lines from Skyrim.ini: %1%'
+          # Other languages omitted for brevity
+        subs:
+          - |
+
+            ```
+            [Audio]
+            fMusicDuckingSeconds=6.0
+            fMusicUnDuckingSeconds=8.0
+            fMenuModeFadeOutTime=3.0
+            fMenuModeFadeInTime=1.0
+            ```
+</code></pre>
+
+<p>Plugins that are <i>not</i> identified by the above Regex:</p>
+
+<pre><code>Example 1.esp
+Example 12.esp
+</code></pre>
+
+<p>Plugins that are identified by the above Regex:</p>
+
+<pre><code>Example 123.esp
+Example 1234.esp
+</code></pre>
+
+<p>Summary:</p>
+
+<ul>
+  <li>The period <code>.</code> has been escaped through writing a backslash <code>\</code> in front of it.</li>
+  <li>The expression <code>\d{3,}</code> finds plugins that contain three or more integer numbers.</li>
+  <li>In addition to the regex this example also showcases a method to display blocks of code within a message for plugins.</li>
+  <li>The mentioned <code>Skyrim.ini</code> can be found in <code>C:\Users\USER\Documents\My Games\Skyrim</code></li>
+</ul>
+
+
+<h3>x to y times: <code>{x,y}</code></h3>
```

Comment:
It would be good to explicitly state that this range is inclusive (though it's explained in the example summary).

Reply:
Done in https://github.com/loot/loot.github.io/pull/109/commits/93dc3c9a604aa8a1f77bc25aadf87d808df1b1d9.